test/integration-php/php-amqp/get-test.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+define("HOST",     getenv("TEST_RABBITMQ_HOST") ? getenv("TEST_RABBITMQ_HOST") : "localhost");
+define("PORT",     getenv("TEST_RABBITMQ_PORT") ? getenv("TEST_RABBITMQ_PORT") : "5672");
+define("USER",     getenv("TEST_RABBITMQ_USER") ? getenv("TEST_RABBITMQ_USER") : "guest");
+define("PASS",     getenv("TEST_RABBITMQ_PASS") ? getenv("TEST_RABBITMQ_PASS") : "guest");
+define("ATTEMPTS", getenv("TEST_ATTEMPTS")      ? getenv("TEST_ATTEMPTS")      : 10);
```

Comment:
```suggestion
define("HOST",     getenv("TEST_RABBITMQ_HOST") ?: "localhost");
define("PORT",     getenv("TEST_RABBITMQ_PORT") ?: "5672");
define("USER",     getenv("TEST_RABBITMQ_USER") ?: "guest");
define("PASS",     getenv("TEST_RABBITMQ_PASS") ?: "guest");
define("ATTEMPTS", getenv("TEST_ATTEMPTS")      ?: 10);
```
This should be equivalent, I think.

Reply:
Indeed it is, thanks